system/views.py

```diff
@@ -157,9 +159,29 @@ def post(self, request):
             connection_list,
         )
 
-        args = {'form':form, 'patient':patient, 'visit_connection_list':visit_connection_list}
+        args = {'form': form, 'patient': patient, 'visit_connection_list': visit_connection_list}
         return render(request, self.template_name, args)
 
+
+class UserLogin(LoginView):
+    template_name = 'login.html'
+
+    def post(self, request):
+        username = request.POST['username']
+        password = request.POST['password']
+        user = authenticate(username=username, password=password)
+
+        if user is not None:
+            if user.is_active:
+                login(request, user)
+
+                return HttpResponseRedirect('/system/view_patients')
+            else:
+                return HttpResponse("Inactive user.")
```

Comment:
Maybe just redirect to `/` if user is not interactive? Showing `Inactive User` may not be clear about next steps.

Reply:
Other than this I think LGTM.

Reply:
Quick explanation to others: LGTM - Looks Good To Me
:)